tracer/test/Datadog.Trace.IntegrationTestHelpers/AssemblyInfo.cs

```diff
@@ -0,0 +1,10 @@
+// <copyright file="AssemblyInfo.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Datadog.Trace.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010025b855c8bc41b1d47e777fc247392999ca6f553cdb030fac8e3bd010171ded9982540d988553935f44f7dd58cb4b17fbb92653d5c2dc5112696886665b317c6f92795bf64beab2405c501c8a30cb1b31b1541ed66e27d9823169ec2815b00ceeeecc8d5a1bf43db67d2961a3e9bea1397f043ec07491709649252f5565b756c5")]
```

Comment:
It would be great, but it's not really possible because the helper have a quite a lot of mutual dependences, particularly `MockTracerAgent` is used across all tests and in a lot of different helpers.

Reply:
Changed how the VerifyHelper is shared between the projects and removed `Datadog.Trace.IntegrationTesthelpers`